src/components/VideoItem/VideoItem.styles.css

```diff
@@ -0,0 +1,41 @@
+.column {
+    float: left;
+    padding: 0 10px;
+    margin: 25px 0 0 0;
+}
+
+.card {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    padding: 16px;
+    text-align: center;
+    height: 400px;
+    width: 350px;
+    cursor: pointer;
+}
+
+.card:hover{
+    background-color: rgb(201, 201, 201)!important;
```

Comment:
Avoid using importants, instead of this try to attach more specificity to your components. 
https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity